src/utils/__tests__/makeBackoffMachine-test.js

```diff
@@ -2,15 +2,31 @@
 import { makeBackoffMachine } from '../async';
 
 describe('makeBackoffMachine', () => {
-  test('first timeout is 0 seconds, then increase by 2 seconds', async () => {
-    const start = Date.now();
-    await makeBackoffMachine();
-    const duration = Date.now() - start;
-    expect(duration).toBeLessThan(1000);
+  test('timeouts are ~100ms, ~200ms, ~400ms, ~800ms', async () => {
```

Comment:
This would definitely be safer with Lolex. (Ref: #3727.)

There should probably also be tests for the backoff machine's auxiliary functions here.

Reply:
Sure; awaiting resolution of https://github.com/zulip/zulip-mobile/pull/3727.